src/react-apps/applications/runtime/src/utils/convertLangToJson.ts

```diff
@@ -0,0 +1,6 @@
+// Is to be executed with node in terminal to convert language
```

Comment:
Do we really need this as part of the runtime source code?

Reply:
Sure.